src/enable_variable_ops/ngraph_utils.cc

```diff
@@ -0,0 +1,326 @@
+/*******************************************************************************
+ * Copyright 2017-2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+#include "tensorflow/core/common_runtime/dma_helper.h"
+#include "tensorflow/core/common_runtime/optimization_registry.h"
+#include "tensorflow/core/framework/attr_value_util.h"
+#include "tensorflow/core/framework/graph.pb.h"
+#include "tensorflow/core/framework/node_def_util.h"
+#include "tensorflow/core/graph/graph.h"
+#include "tensorflow/core/lib/strings/str_util.h"
+#include "tensorflow/core/platform/default/logging.h"
+#include "tensorflow/core/platform/protobuf.h"
+
+#include "ngraph_utils.h"
+using namespace std;
+namespace ng = ngraph;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+// static int testing_graph_id=30;
+
+Status IsCopyLogEnabled(int graph_id, bool& is_copy_log_enabled) {
+  const char* copy_env_var = std::getenv("NGRAPH_TF_LOG_COPIES");
+  if (copy_env_var == nullptr) return Status::OK();
```

Comment:
this function is designed to work in a way that requires the caller to set `is_copy_log_enabled=false;` before calling. If is_copy_log_enabled is set to true, and `NGRAPH_TF_LOG_COPIES` is not set, then this function will keep `is_copy_log_enabled` true.

maybe lets set it to false before returning at this point

Reply:
Done